hugegraph-server/hugegraph-dist/src/assembly/static/bin/init-store.sh

```
@@ -48,13 +48,14 @@ cd "${TOP}" || exit
 DEFAULT_JAVA_OPTIONS=""
 JAVA_VERSION=$($JAVA -version 2>&1 | awk 'NR==1{gsub(/"/,""); print $3}' | awk -F'_' '{print $1}')
```

Shellcheck (0.11.0)

[warning] JAVA_VERSION appears unused. Verify use (or export if used externally).

(SC2034)